src/autorenderloop.js

```diff
@@ -143,9 +143,16 @@ olcs.AutoRenderLoop.prototype.enable = function() {
     that.notifyRepaintRequired();
   };
 
-  // Listen for changes on the layer group
-  this.ol3d.getOlMap().getLayerGroup().on('change',
+  // Listen for changes on the main layer group
+  this.ol3d.getOlMap().getLayerGroup().on('change:layers',
       this._boundNotifyRepaintRequired);
+
+  this.ol3d.getOlMap().getLayers().forEach((layer) => {
+    // Listen for changes on layer group
+    layer.on('change', that._boundNotifyRepaintRequired);
+    // Listen for property changes on simple layer (opacity, visibility, ...)
+    layer.on('propertychange', that._boundNotifyRepaintRequired);
+  });
```

Comment:
@fredj, do you know if the optimization at https://github.com/openlayers/openlayers/blob/v4.2.0/src/ol/layer/group.js#L83 is necessary? Maybe it could be acceptable to relax this condition for the main group?